stubs/inertia-common/tailwind.config.js

```diff
@@ -1,7 +1,8 @@
-const defaultTheme = require('tailwindcss/defaultTheme');
+import defaultTheme from 'tailwindcss/defaultTheme';
+import forms from '@tailwindcss/forms'
```

Comment:
📌 `'tailwindcss/defaultTheme'` does not require, or work with, a prefixed `@` here